.pre-commit-config.yaml

```diff
@@ -3,17 +3,3 @@
   hooks:
     - id: go-fmt
     - id: go-lint
-    - id: validate-toml
-    - id: no-go-testing
-- repo: https://github.com/wangkuiyi/google-style-precommit-hook
-  rev: v0.1.1
-  hooks:
-    - id: google-style-java
-      pass_filenames: false # https://github.com/sql-machine-learning/sqlflow/issues/1152#issuecomment-551325589
-- repo: local
-  hooks:
-    - id: copyright_checker
-      name: copyright_checker
-      entry: python ./scripts/copyright.py
```

Comment:
1. `copyright.py` adds copyright information. I think we need it
2. If you remove the `copyright_checker`, please remove the `scripts/copyright.py` either.

Reply:
Good catch. I have added back the copyright checker.